### PR TITLE
Business methods at one place pair lp factory

### PR DESCRIPTION
**Pull request by shivamgarglab49**

### CODE REVIEW

File: e2e-test/business/Common.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  PrivateKey,
+  TokenInfoQuery,
+  TokenSupplyType,
+  AccountBalanceQuery,
+  TokenCreateTransaction,
+} from "@hashgraph/sdk";
+import { BigNumber } from "bignumber.js";
+import { clientsInfo } from "../../utils/ClientManagement";
+
+export default class Common {
```
Comment (amandeeprehal-lab49) [inline e2e-test/business/Common.ts:16]: They are public right, pls add modifier.

File: e2e-test/business/Factory.ts [hunk 2]
```diff
@@ -1,176 +1,121 @@
...
-    client: Client
+export default class Factory extends Base {
+  private contractId: string;
+
+  constructor(_contractId: string) {
+    super();
+    this.contractId = _contractId;
+  }
+
+  setupFactory = async (
```
Comment (amandeeprehal-lab49) [inline e2e-test/business/Factory.ts:30]: Same here

File: e2e-test/business/LpToken.ts [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+export default class LpToken extends Base {
+  contractId: string;
+
+  constructor(_contractId: string) {
+    super();
+    this.contractId = _contractId;
+  }
+
+  initialize = async (
```
Comment (amandeeprehal-lab49) [inline e2e-test/business/LpToken.ts:26]: Same

File: e2e-test/business/Common.ts (hunk 1 above)
Comment (shivamgarglab49) [reply e2e-test/business/Common.ts:16]: they are public by default.

**Pull request closed without merging**